doc/api/mruby/string.h.md

```diff
@@ -43,3 +47,45 @@ Duplicates a string object.
    mrb_value mrb_str_intern(mrb_state *mrb, mrb_value self);
 ```
 Returns a symbol from a passed in string.
+### mrb_str_to_str
+```C
+   mrb_value mrb_str_to_str(mrb_state *mrb, mrb_value str);
+```
+Returns a converted string type.
+### mrb_str_equal
+```C
+   mrb_bool mrb_str_equal(mrb_state *mrb, mrb_value str1, mrb_value str2);
+```
+Returns true if the strings match and false if the strings don't match.
+### mrb_str_cat
+```C
+   mrb_value mrb_str_cat(mrb_state *mrb, mrb_value str, const char *ptr, size_t len);
+```
+Returns a concated string comprised of a Ruby string and a C string.
+### mrb_str_cat_cstr
+```C
+   mrb_value mrb_str_cat_str(mrb_state *mrb, mrb_value str, mrb_value str2);
+```
+Returns a concated string comprised of a Ruby string and a C string(A shorter alternative to mrb_str_cat).
+### mrb_str_append
+```C
+   mrb_value mrb_str_append(mrb_state *mrb, mrb_value str, mrb_value str2);
+```
+Adds str2 to the end of str1.

```

Comment:
Ok yeah, i was looking at the implementation :-)



—
Sent from Mailbox

On Wed, Sep 2, 2015 at 5:15 AM, Ralph Desir <notifications@github.com>
wrote:

>> +Returns true if the strings match and false if the strings don't match.
>> +### mrb_str_cat
>> +```C
>> +   mrb_value mrb_str_cat(mrb_state *mrb, mrb_value str, const char *ptr, size_t len);
>> +```
>> +Returns a concated string comprised of a Ruby string and a C string.
>> +### mrb_str_cat_cstr
>> +```C
>> +   mrb_value mrb_str_cat_str(mrb_state *mrb, mrb_value str, mrb_value str2);
>> +```
>> +Returns a concated string comprised of a Ruby string and a C string(A shorter alternative to mrb_str_cat).
>> +### mrb_str_append
>> +```C
>> +   mrb_value mrb_str_append(mrb_state *mrb, mrb_value str, mrb_value str2);
>> +```
>> +Adds str2 to the end of str1.
> It's str1 in the header file.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mruby/mruby/pull/2931/files#r38495815